### PR TITLE
fix #161 add function for the user to restore autosaved new pages

### LINKED ISSUE #161: restore new autosaved sites
add an easy-to-use functionality to restore sites that have only been autosaved without using the admin panel

### PR DESCRIPTION
**Pull request by Nef10**

### CODE REVIEW

File: _1327/documents/templates/editor.html [hunk 1]
```diff
@@ -8,11 +8,15 @@
 		<div class="alert alert-danger alert-margin-bottom" role="alert">{% trans "Please provide the missing information" %}</div>
 	{% endif %}
 
-    {% if form.autosave %}
-		<div class="alert alert-danger alert-margin-bottom" role="alert">{% trans "This document was autosaved on" %} {{ form.autosave_date }}{% trans ", you can restore unsaved changes:" %} <a href="{{ edit_url }}?restore" class="btn-sm btn-default" role="button">{% trans "Click here" %}</a></div>
+	{% if form.autosave %}
+		<div class="alert alert-danger alert-margin-bottom" role="alert">{% trans "This document was autosaved on" %} {{ form.autosave_date }}{% trans ", you can restore unsaved changes:" %} <a href="{% url 'information_pages:edit' document.url_title %}?restore" class="btn-sm btn-default" role="button">{% trans "Click here" %}</a></div>
```
Comment (invliD) [inline _1327/documents/templates/editor.html]: Don't use information page directly! The ``document`` app should not know about derived apps.

File: _1327/documents/templates/editor.html [hunk 2]
```diff
@@ -8,11 +8,15 @@
...
 	{% endif %}
-	
-	<form class="form-horizontal" action="{{ edit_url }}" method="post" role="form" id="document-form">
+
+	{% for new_autosaved_page in new_autosaved_pages %}
+		<div class="alert alert-danger alert-margin-bottom" role="alert">{% trans "There is a new document autosaved on" %} {{ new_autosaved_page.created }}{% trans ", you can restore this unsaved page:" %} <a href="{% url 'information_pages:edit' new_autosaved_page.url_title %}?restore" class="btn-sm btn-default" role="button">{% trans "Click here" %}</a></div>
+	{%  endfor %}
+
+	<form class="form-horizontal" action="{% url 'information_pages:edit' document.url_title %}" method="post" role="form" id="document-form">
```
Comment (invliD) [inline _1327/documents/templates/editor.html]: See above (https://github.com/fsr-itse/1327/pull/164/files#diff-eeca56535cbdaf4c9aeb4a69312da96bR12)

**Comment by janno42** [comment]
:+1:

**Comment by invliD** [comment]
:+1:

**Pull request merged**